packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts

```diff
@@ -162,6 +162,15 @@ export class LocalTunnelTaskTerminal extends BaseTaskTerminal {
       ? [this.args.ngrokArgs]
       : this.args.ngrokArgs;
 
+    if (isV3Enabled()) {
+      if (!this.args.output?.domain) {
```

Comment:
Do we support no output? E.g., just want to start ngrok but not want to output anything.

Reply:
Got it, update the args va;idation logic.